### DISCUSSION TITLE
The visualisation of the tag characters and  \XeTeXglyph

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#300] [Q&A]
The editor of a paper of mine has some problem with typesetting the visualisation of the tag characters. I hoped they can be access directly with \XeTeXglyph but it seems I'm wrong. "a tag" is at slot 1114605, but \XeTeXglyph does not accept such an argument. Any suggestions?

**Comment by psb1558** [comment]
The Unicode tag characters have the type "default ignorable," so they are invisible and zero-width, like (for example) the zero-width non-joiner. That makes them hard to manage in word processors, so Junicode has a parallel set of visible tags. These have no code points but instead are accessible via entities consisting of an ampersand, two underscores, the tag identifier, and a semicolon. So `&__s;` gives you the visible tag character corresponding to `\char"0E0073`:
<img width="64" alt="image" src="https://github.com/user-attachments/assets/e8ab4888-6a31-4356-bee7-519a9e79566b">
Apply `ss10` to this to convert the visible tag to the invisible Unicode tag and resolve the character+tag+tag sequences listed in the Junicode Manual Chapter 5.

My thought, when setting up this system, was that users could use the visible tags for ease of editing (not generally necessary but perfectly okay in XeTeX), and then apply `ss10` to the whole document for presentation.

In sum: instead of `\char"0E0073`, use `&__s;` **without** `ss10` to get the visible tag.

**Reply by jsbien** [reply]
I know this and this works for me but for unknown reasons wasn't working elsewhere. The editor is quite fluent in font tools, so he circumvented the problem by extracting the required glyphs from Junicode into a new font. From my point of view the problem is solved.

**Reply by psb1558** [reply]
I'm glad your editor found a workaround, but it sounds like an awkward one. Problems with Junicode are often caused by an old version of the font hanging around in the TeX installation. If not, and your editor is willing to share his code, I can probably get it working the way it should. I've never had any difficulty with these tags.

**Reply by jsbien** [reply]
I also never had problems with the tags.
His setup is quite complicated. I tried to reproduce the problem but gave up. He uses its own document class, which he provided to me, but uses also a lot of some fonts I'm not familiar with. At present his priority  is to ship the book to the printer. Perhaps later he will be interested in investigating the problem.
BTW, recent Emacs has glyphless-display-mode which looks quite useful.

**Comment by jsbien** [comment]
FYI, I accepted a brute force work-arounds in the official paper https://doi.org/10.36824/2022-graf-bien and made a properly typeset postprint https://www.researchgate.net/publication/390326449_16th_century_Latin_printed_brevigraphs_in_Unicode---a_computer_resource.

**Reply by psb1558** [reply]
Both versions came out looking good!

**Reply by jsbien** [reply]
Thanks!
BTW, I tried to close the issue as outdated, but it doesn't work.

**Comment by psb1558** [comment]
I'll try it.

**Comment by psb1558** [comment]
Doesn't work for me either. I suppose it wants an answer marked as correct.
